src/assets/styles/_variables.scss

```diff
@@ -30,8 +30,6 @@
   // KONGPONENTS
   // Reduces the excessive vertical padding.
   --KPopPaddingY: var(--spacing-xs);
-  // Sets the font size to the size of body copy as the default of 12 pixels is way too small.
-  --KBadgeFontSize: 1rem;
```

Comment:
**Issue**: Following the comment about not using non-relative units for the body font size, this shouldn’t be removed.

Reply:
See above